### PR TITLE
cert-rotation: replace wait for Progressing=True in wait-for-kubeapiserver-to-start-progressing

### PR DESCRIPTION
**Pull request by vrutkovs**
Use sleep 5m instead as kube-apiserver operator doesn't report
progressing correctly

### CODE REVIEW

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.19-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d periodic-ci-openshift-release-master-nightly-4.17-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d

**Comment by vrutkovs** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.19-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d periodic-ci-openshift-release-master-nightly-4.18-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d periodic-ci-openshift-release-master-nightly-4.17-e2e-metal-ovn-sno-cert-rotation-shutdown-180d-age-90d

File: ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh [hunk 1]
```diff
@@ -121,7 +121,9 @@ function wait-for-valid-lb-ext-kubeconfig {
 cat << 'EOZ' > /tmp/wait-for-kubeapiserver-to-start-progressing.sh
   echo "Waiting for kube-apiserver to start progressing to avoid stale operator statuses"
   export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/lb-ext.kubeconfig
-  oc --request-timeout=5s wait --for=condition=Progressing clusteroperator/kube-apiserver --timeout=300s 
+  # TODO: kube-apiserver never starts progressing in 4.19+
+  # oc --request-timeout=5s wait --for=condition=Progressing=True clusteroperator/kube-apiserver --timeout=300s
```
Comment (wangke19) [inline ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh:125]: Is this a bug? In QE CI 4.19 tests, we don't see this.

Comment (vrutkovs) [reply ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh:125]: This seems to happen only when instance is being restarted after shutdown/suspend. I'd like to investigate this more later and land the fix to unblock testing now

Comment (wangke19) [reply ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh:125]: Okay. 
/lgtm

**Pull request merged**